app/imports/components/comments/NewComment.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+
+import sanitizeHtml from 'sanitize-html';
+
+import { addComment } from '/imports/api/commentsMethods.js';
+
+export class NewComment extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onSubmitForm = this.onSubmitForm.bind(this);
+    this.clearInput = this.clearInput.bind(this);
+  }
+
+  onInputChange(event) {
+    this.setState({comment: event.target.value});
+  }
+
+  clearInput() {
+    this.setState({comment: ''});
+  }
+
+  onSubmitForm(event) {
+    event.preventDefault();
+
+    const comment = sanitizeHtml(this.state.comment);
+
+    addComment.call({
```

Comment:
there should be a check for the comment being empty after sanitization otherwise it is possible to create comments that are just `""`